nav2_navfn_planner/src/navfn_planner.cpp

```diff
@@ -320,7 +320,9 @@ NavfnPlanner::getPlanFromPotential(
 
   planner_->setStart(map_goal);
 
-  int path_len = planner_->calcPath(costmap_->getSizeInCellsX() * 4);
+  const int& max_cycles = (costmap_->getSizeInCellsX() >= costmap_->getSizeInCellsY()) ? (costmap_->getSizeInCellsX() * 4) : (costmap_->getSizeInCellsY() * 4);
```

Comment:
`int &`, the linters are going to complain. You can test this locally with `ament_cpplint` and `ament_uncrustify`